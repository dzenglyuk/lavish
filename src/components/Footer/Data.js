export const infoLinks = [
    {
        title: "About Us",
        links: [
            {
                name: "How it works",
                value: "/sign-up",
            },
            {
                name: "Testimonials",
                value: "/",
            },
            {
                name: "Careers",
                value: "/",
            },
            {
                name: "Investors",
                value: "/",
            },
            {
                name: "Terms of Service",
                value: "/",
            },
        ],
    },
    {
        title: "Contact Us",
        links: [
            {
                name: "Contact",
                value: "/",
            },
            {
                name: "Support",
                value: "/",
            },
            {
                name: "Destinations",
                value: "/",
            },
            {
                name: "Sponsorships",
                value: "/",
            },
        ],
    },
];

export const socialLinks = [
    {
        title: "Videos",
        links: [
            {
                name: "Submit Video",
                value: "/",
            },
            {
                name: "Ambassadors",
                value: "/",
            },
            {
                name: "Agency",
                value: "/",
            },
            {
                name: "Influencer",
                value: "/",
            },
        ],
    },
    {
        title: "Social Media",
        links: [
            {
                name: "Instagram",
                value: "/",
            },
            {
                name: "Facebook",
                value: "/",
            },
            {
                name: "Youtube",
                value: "/",
            },
            {
                name: "Twitter",
                value: "/",
            },
        ],
    },
];

export const subscription = {
    heading: "Join our exclusive membership to receive the latest news and trends",
    text: "You can unsubscribe at any time."
};