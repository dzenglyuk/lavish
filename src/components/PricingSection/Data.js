const starterAccount = {
    info: {
        name: "Starter",
        price: "$8.99",
        period: "per month"
    },
    features: ["100 Transactions", "2% Cash Back", "$10,000 Limit"]
};

const goldAccount = {
    info: {
        name: "Gold",
        price: "$29.99",
        period: "per month"
    },
    features: ["1000 Transactions", "3.5% Cash Back", "$100,000 Limit"]
};

const diamondAccount = {
    info: {
        name: "Diamond",
        price: "$99.99",
        period: "per month"
    },
    features: ["Unlimited Transactions", "5% Cash Back", "Unlimited Spending"]
};

export {
    starterAccount,
    goldAccount,
    diamondAccount
};