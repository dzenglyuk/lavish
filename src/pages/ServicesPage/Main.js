import React from "react";
import HomeSection from "../../components/HomeSection/Main";
import PricingSection from "../../components/PricingSection/Main";
import { homeObjFour } from "../Data"; 

const Main = () => {
    return (
        <React.Fragment>
            <PricingSection />
            <HomeSection {...homeObjFour} />
        </React.Fragment>
    )
}

export default Main;
