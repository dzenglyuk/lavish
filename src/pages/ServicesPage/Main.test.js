import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import { StaticRouter as Router } from "react-router";

import { homeObjFour } from "../Data";
import ServicesPage from "./Main";

test("Renders correctly", () => {
    const { getByText } = render(
        <Router>
            <ServicesPage />
        </Router>
    );
    getByText(homeObjFour.headLine);
    getByText(homeObjFour.topLine);
    getByText("Pricing");
    getByText("Starter");
    getByText("Gold");
    getByText("Diamond");  
});

test("Matches the snapshot", () => {
    const tree = renderer
        .create(
            <Router>
                <ServicesPage />
            </Router>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
