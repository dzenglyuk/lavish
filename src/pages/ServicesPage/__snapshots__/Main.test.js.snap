// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
Array [
  <div>
    <div
      className="pricing__section"
    >
      <div
        className="pricing__wrapper"
      >
        <h1
          className="pricing__heading"
        >
           
          Pricing
           
        </h1>
        <div
          className="pricing__container"
        >
          <a
            className="pricing__container-card"
            href="/sign-up"
            onClick={[Function]}
          >
            <div
              className="pricing__container-cardInfo"
            >
              <div
                className="icon"
              >
                <svg
                  fill="currentColor"
                  height={64}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "#fff",
                    }
                  }
                  viewBox="0 0 384 512"
                  width={64}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M216 23.86c0-23.8-30.65-32.77-44.15-13.04C48 191.85 224 200 224 288c0 35.63-29.11 64.46-64.85 63.99-35.17-.45-63.15-29.77-63.15-64.94v-85.51c0-21.7-26.47-32.23-41.43-16.5C27.8 213.16 0 261.33 0 320c0 105.87 86.13 192 192 192s192-86.13 192-192c0-170.29-168-193-168-296.14z"
                  />
                </svg>
              </div>
              <h3>
                 
                Starter
                 
              </h3>
              <h4>
                 
                $8.99
                 
              </h4>
              <p>
                 
                per month
                 
              </p>
              <ul
                className="pricing__container-features"
              >
                <li>
                  100 Transactions
                </li>
                <li>
                  2% Cash Back
                </li>
                <li>
                  $10,000 Limit
                </li>
              </ul>
              <button
                className="btn 
			btn--primary
			btn--wide
			primary"
              >
                Choose Plan
              </button>
            </div>
          </a>
          <a
            className="pricing__container-card"
            href="/sign-up"
            onClick={[Function]}
          >
            <div
              className="pricing__container-cardInfo"
            >
              <div
                className="icon"
              >
                <svg
                  fill="currentColor"
                  height={64}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "#fff",
                    }
                  }
                  viewBox="0 0 16 16"
                  width={64}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clipRule="evenodd"
                    d="M9.05.435c-.58-.58-1.52-.58-2.1 0L4.047 3.339 8 7.293l3.954-3.954L9.049.435zm3.61 3.611L8.708 8l3.954 3.954 2.904-2.905c.58-.58.58-1.519 0-2.098l-2.904-2.905zm-.706 8.615L8 8.707l-3.954 3.954 2.905 2.904c.58.58 1.519.58 2.098 0l2.905-2.904zm-8.615-.707L7.293 8 3.339 4.046.435 6.951c-.58.58-.58 1.519 0 2.098l2.904 2.905z"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
              <h3>
                 
                Gold
                 
              </h3>
              <h4>
                 
                $29.99
                 
              </h4>
              <p>
                 
                per month
                 
              </p>
              <ul
                className="pricing__container-features"
              >
                <li>
                  1000 Transactions
                </li>
                <li>
                  3.5% Cash Back
                </li>
                <li>
                  $100,000 Limit
                </li>
              </ul>
              <button
                className="btn 
			btn--primary
			btn--wide
			blue"
              >
                Choose Plan
              </button>
            </div>
          </a>
          <a
            className="pricing__container-card"
            href="/sign-up"
            onClick={[Function]}
          >
            <div
              className="pricing__container-cardInfo"
            >
              <div
                className="icon"
              >
                <svg
                  fill="currentColor"
                  height={64}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "#fff",
                    }
                  }
                  viewBox="0 0 512 512"
                  width={64}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M246.785 37.605l-2.242 6.14s-17.637 48.195-43.14 94.23c-25.83 46.624-27.81 45.724-59.56 91.02-21.79 31.088-45.966 65.665-45.966 110.42 0 86.952 70.57 158.216 157.445 158.218 86.877.002 157.446-71.266 157.444-158.22 0-44.647-24.275-79.385-45.955-110.407-31.968-45.738-34.803-46.012-60.623-92.617C278.693 90.37 262.13 43.82 262.13 43.82l-2.21-6.217h-13.135zm14.553 53.91c6.923 15.817 15.76 34.54 26.502 53.93 2.427 4.382 4.61 8.263 6.693 11.932-8.75-5.492-17.206-7.09-23.43-3.498-12.884 7.438-11.325 34.263 3.483 59.91 14.808 25.65 37.26 40.412 50.144 32.974 7.982-4.61 10.415-16.658 7.596-31.256 4.793 6.682 10.236 14.287 17.166 24.203 3.74 5.353 7.452 10.707 11.047 16.095l-32.55 28.63-96.86-48.087 30.208-144.834zm-21.535 11.717L215.457 219.96c-2.022-3.156-4.77-4.95-8.22-5.028-13.51-.304-32.587 25.708-42.6 58.093-10.013 32.386-7.177 58.89 6.336 59.194 3.61.08 7.62-1.734 11.748-5.033l-7.93 25.036-59.503-24.89c3.816-32.54 22.478-59.962 41.858-87.61 31.478-44.904 34.522-45.61 60.605-92.692 8.453-15.256 15.815-30.24 22.053-43.798zm-14.412 151.13l41.05 20.382c-23.878 4.858-41.848 25.972-41.848 51.287 0 28.91 23.433 52.343 52.34 52.343 15.083 0 28.667-6.387 38.218-16.596l-3.572 32.907-76.04 22.18-44.308-54.653 34.16-107.847zm145.258 17.44c4.188 7.078 8.015 14.266 11.22 21.653-1.008-.193-2.032-.305-3.073-.305-14.32 0-25.932 18.828-25.932 42.047 0 23.22 11.61 42.047 25.932 42.047 3.078 0 6.03-.876 8.77-2.473-3.75 14.402-9.734 27.906-17.56 40.11l-39.808-18.534 10.674-98.352 29.778-26.19zm-255.85 75.587l49.354 20.64c-9.514 3.844-16.23 13.162-16.23 24.052 0 14.32 11.61 25.932 25.932 25.932 11.32 0 20.94-7.258 24.478-17.37l21.62 26.667-8.634 45.104c-53.702-17.164-93.218-66.33-96.52-125.025zm219.294 71.383l24.54 11.428c-25.442 29.812-63.17 48.744-105.31 48.743-8.128 0-16.09-.71-23.834-2.062l7.867-41.08 30.368-8.857c-6.227 4.895-10.264 10.065-10.762 14.573-1.096 9.93 15.284 12.87 36.585 6.57 21.246-6.285 39.364-19.39 40.547-29.313z"
                  />
                </svg>
              </div>
              <h3>
                 
                Diamond
                 
              </h3>
              <h4>
                 
                $99.99
                 
              </h4>
              <p>
                 
                per month
                 
              </p>
              <ul
                className="pricing__container-features"
              >
                <li>
                  Unlimited Transactions
                </li>
                <li>
                  5% Cash Back
                </li>
                <li>
                  Unlimited Spending
                </li>
              </ul>
              <button
                className="btn 
			btn--primary
			btn--wide
			primary"
              >
                Choose Plan
              </button>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="home__section darkBg"
  >
    <div
      className="container"
    >
      <div
        className="row home__home-row"
        style={
          Object {
            "display": "flex",
            "flexDirection": "row-reverse",
          }
        }
      >
        <div
          className="col"
        >
          <div
            className="home__text-wrapper"
          >
            <div
              className="top-line"
            >
              DATA ANALYTICS
            </div>
            <h1
              className="heading"
            >
              Every transaction is stored on our secure cloud database
            </h1>
            <p
              className="home__subtitle"
            >
               
              Never ever have to worry again about saved reciepts. We store your data, so you can access it anytime.
               
            </p>
            <a
              href="/sign-up"
              onClick={[Function]}
            >
              <button
                className="btn 
			btn--primary
			btn--wide
			blue"
              >
                Sign Up Now
              </button>
            </a>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="home__img-wrapper"
          >
            <img
              alt="Vault"
              className="home__img"
              src="images/svg-8.svg"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
